### PR TITLE
Update cw-plus packages to 0.15.0

### PR DESCRIPTION
**Pull request by ueco-jb**

### CODE REVIEW

**Comment by ethanfrey** [comment]
Oh great,  cw-nfts may be one of the first one to update. Cutting edge. 😄 

Looking forward to this. (It provides much better schemas and will make ts-codegen happy)

**Comment by JakeHartnell** [comment]
Going to need a rebase. :joy:

**Comment by ethanfrey** [comment]
You just did a big merge?
So we can get them all updated?

File: contracts/cw721-base/schema/approvals_response.json [hunk 1]
```diff
@@ -76,7 +76,8 @@
           ],
           "properties": {
             "never": {
-              "type": "object"
+              "type": "object",
+              "additionalProperties": false
```
Comment (ethanfrey) [inline contracts/cw721-base/schema/approvals_response.json:80]: nice!

**Review by ethanfrey — DISMISSED** [review]
What I see looks good.

What I don't see are the changes to `schema.rs` and moving them to the bin directory and such. I guess the movement is optional, but I think a big part of the upgrade is also using these unified schema output (the new format from Tom).

I am happy to approve and merge this as is. *But* I would wait until those schema.rs files are modified to tag the repo and make a v0.15 release

**Pull request merged**